Slim/Plugin/Podcast/Parser.pm

```diff
@@ -103,15 +83,14 @@ sub parse {
 				enclosure => {
 					type   => $enclosure->{type},
 					length => $enclosure->{length},
-					url    => $url,
+					url    => $enclosure->{url},
 				},
-				duration => $item->{duration},
-				play => sub {
-						my ($client, $cb) = @_;
-						$client->pluginData(goto => 1);
-						delete $item->{items}->[0]->{play};
-						$cb->( $item->{items}->[0] );
-					},
+				url => sub { 
+					my ($client, $cb) = @_;
+					$client->pluginData(goto => 1);
+					$cb->( $item->{items}->[0] );
+				},	
+				#duration => $item->{duration},				
```

Comment:
please remove if we really don't need it any more

Reply:
I was looking for you direction there. The comment that I've not added is that when you add duration, there is an "all songs" entry being created automatically (I verified why in Web::XMLBrowser) and it's funny to have that when the 2 choices are "play from beginning" or "play from last position". But then we lose "duration" when we click on info... although it only happens when a track has been partly played

Reply:
I could live with that missing feature.

Reply:
done